angular-rails-app/app/assets/javascripts/property/property.html

```diff
@@ -1,8 +1,7 @@
 <section style="min-height: 65px;margin-top:105px;">
   <div class="container">
     <p id="breadcrumb">
-      <a class="breadcrumb_top" ui-sref="home">Home /</a> Building /
-      {{vm.data.street_address | titleCase}}, {{vm.data.city}} {{vm.data.state}}
+      <a class="breadcrumb_top" ui-sref="home">Home /</a> Building 
```

Comment:
Yeah.... the CSS needs to be cleaned up on this page.  :)  What is 30px though?  I'm not sure which part you are referring to..?